guidelines/releases/release-process.md

```diff
@@ -172,13 +192,15 @@ Follow these steps to create a release:
 7. After all checks pass, merge the PR.
     > **NOTE:** To merge the PR to the release branch, you must receive approvals from all teams.
 
-8. Merging the PR to the release branch runs the postsubmit job that creates a GitHub release.
+8. Merging the PR to the release branch runs the postsubmit job that creates a GitHub release. This job also triggers documentation update on the official Kyma website.
+
+9. Update `RELEASE_VERSION` to the next patch RC1 version on the release branch. Do it immediately after the release, otherwise any PR to a release branch overrides the previously published Docker images.
```

Comment:
I don't get the next patch RC1 version.

Reply:
yes, it always refer to a file.

Reply:
so
```suggestion
9. Update the `RELEASE_VERSION` file to contain the next patch RC1 version on the release branch. Do it immediately after the release, otherwise, any PR to a release branch overrides the previously published Docker images.
```